com.unity.render-pipelines.high-definition/Runtime/Material/StackLit/StackLit.hlsl

```diff
@@ -1200,6 +1196,25 @@ void ClampRoughness(inout PreLightData preLightData, inout BSDFData bsdfData, fl
     }
 }
 
+void GetVLayeredBottomLobesPerceptualRoughnesses(BSDFData bsdfData, PreLightData preLightData, out float basePerceptualRoughness[BASE_NB_LOBES])
+{
+    bool perLightOption = GetRecomputeStackPerLightOption();
+    bool haveAnisotropy = HasFlag(bsdfData.materialFeatures, MATERIALFEATUREFLAGS_STACK_LIT_ANISOTROPY);
+
+    if (perLightOption && haveAnisotropy)
+    {
+        basePerceptualRoughness[0] = preLightData.unmodifiedIblPerceptualRoughness[0];
```

Comment:
Maybe use indices to reference coats when in relative space (when cannot be using BASE_LOBEA_IDX and BASE_LOBEB_IDX)? a bit confusing to follow

Reply:
Yeah I've considered it since the beginning when I made these indices (between having yet another set for BASE_NB_LOBES sized arrays or just or 0-1), right now in the code everywhere where we process only BASE_NB_LOBES it is 0,1 so this addition is in line with that, but I can change it everywhere np.